production/helm/loki/values.yaml

```
@@ -156,6 +157,11 @@ loki:
   # Should authentication be enabled
   auth_enabled: true
 
+  # https://grafana.com/docs/loki/latest/configuration/#server
```

Nitpick. The common pattern seems to be
```suggestion
  # -- Check https://grafana.com/docs/loki/latest/configuration/#server for more info on the server configuration.
```